eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -0,0 +1,438 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Use case: This script verifies the following:
+# 1. There are no duplicate entries in any of the version_*.txt files
+# 2. There are no duplicate entries in the external_dependencies.txt file
+# 3. POM file verification across the enlistment which includes the following:
+#    a. There are no <dependencyManagement> sections
+#    a. Every <dependency> and <plugin> has a <groupId>, <artifactId> and <version>
+#    b. Every <version> has the appropriate x-version-update tag
+#    c. The <version>'s value is the same as the value in the version_*txt file or external_dependency
+#
+# Output:
+# This script will process the entire enlistment. If any errors are encountered, will report them at
+# time they are encountered and continue processing.
```

Comment:
enlistment ->repo

"will report them at time they are encountered and continue processing." -> "it will report them at the time they are encountered and continue processing"

Reply:
Already changed and will be updated with the next commit.